test/rollup_node/index.mjs

```diff
@@ -0,0 +1,5 @@
+import Module from './hello.mjs';
+
+Module().then((instance) => {
```

Comment:
could be `const instance = await Module();` ...

Reply:
Sure!